### PR TITLE
Add missing <cmath> header for fabs to fix Visual C++ vNext break, av…

### PR DESCRIPTION
**Pull request by BillyONeal**
…oid narrowing to float from double

### CODE REVIEW

File: tests/src/Interop/COM/NativeClients/Primitives/ArrayTests.cpp [hunk 1]
```diff
@@ -3,6 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 #include "ClientTests.h"
+#include <stdio.h>
```
Comment (AaronRobinsonMSFT) [inline tests/src/Interop/COM/NativeClients/Primitives/ArrayTests.cpp]: Could you instead make that `#include <cstdio>`?

Comment (BillyONeal) [reply tests/src/Interop/COM/NativeClients/Primitives/ArrayTests.cpp]: I can, but this file says `::printf` not `std::printf`, and the standard doesn't require that `<cstdio>` provides that (though in practice I know of no implementation that does not provide it). Do you want those fixed up too?

Comment (AaronRobinsonMSFT) [reply tests/src/Interop/COM/NativeClients/Primitives/ArrayTests.cpp]: oh, that is right - boo for me not following the standard. You can leave this as is. Thanks.

**Pull request merged**